databrowser/src/components/pill/PillGroup.vue

```diff
@@ -1,44 +1,24 @@
 <template>
   <div>
-    <div
+    <PillGroupItem
       v-for="(item, index) in data"
       :key="item"
```

Comment:
What if two values in the array are the same? For example ["2", "2", "1"]. Then the key would be the same. Would it maybe make more sense to use the index for the key? Or do we have there also some problems? For example if the pagination changes and then page 4 moves from index 3 to index 2. I'm not even sure by myself what to choose here.